client/llb/fileop.go

```diff
@@ -726,6 +720,12 @@ func (f *FileOp) Marshal(ctx context.Context, c *Constraints) (digest.Digest, []
 		})
 	}
 
+	pop, md := MarshalConstraints(c, &f.constraints)
+	pop.Op = &pb.Op_File{
+		File: pfo,
+	}
+	pop.Inputs = state.inputs
```

Comment:
This doesn't need special definition row now.

Reply:
> This doesn't need special definition row now.

Sorry, I don't understand. How else would you write this?